ssl_test.go

```diff
@@ -0,0 +1,466 @@
+//go:build !go_tarantool_ssl_disable
```

Comment:
We have a single connection to a single instance and so-called multiconnection. Shouldn't we add a small test for muticonnection with SSL?

Reply:
I don't think so:

Multiconnection is about handling and managing multiple connections. Not about a single connection and it options. It would be too much to test the connection options there (for example, we don't test in multi_test.go other connection options, just multiconnection logic).

Reply:
The connection pool (as well as the deprecated multi API) just passes the options to individual connections, so it looks orthogonal enough to don't insist on a special test. Closing the discussion.